2018-05-30T02:33:12.8815023+03:00 0HLE5NI5N86U8:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/   (ca22a1cb)
2018-05-30T02:33:15.5702569+03:00 0HLE5NI5N86U8:00000001 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T02:33:18.1938367+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("71"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"CountInvites\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T02:33:18.7523595+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("2"ms) [Parameters=["@__user_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"uc\".\"Id\", \"uc\".\"ClaimType\", \"uc\".\"ClaimValue\", \"uc\".\"UserId\"
FROM \"AspNetUserClaims\" AS \"uc\"
WHERE \"uc\".\"UserId\" = @__user_Id_0" (0723d8ff)
2018-05-30T02:33:18.9087428+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("3"ms) [Parameters=["@__userId_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"role\".\"Name\"
FROM \"AspNetUserRoles\" AS \"userRole\"
INNER JOIN \"AspNetRoles\" AS \"role\" ON \"userRole\".\"RoleId\" = \"role\".\"Id\"
WHERE \"userRole\".\"UserId\" = @__userId_0" (0723d8ff)
2018-05-30T02:33:18.9598501+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("8"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T02:33:19.0327045+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T02:33:19.0519732+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T02:33:19.0575161+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T02:33:19.0601019+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("0"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T02:33:19.0629786+03:00 0HLE5NI5N86U8:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T02:33:19.0708555+03:00 0HLE5NI5N86U8:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T02:33:19.4513496+03:00 0HLE5NI5N86U8:00000001 [INF] Executing action method "PeerReview.Controllers.HomeController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T02:33:25.4199726+03:00 0HLE5NI5N86U8:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-05-30T02:33:27.3178159+03:00 0HLE5NI5N86U8:00000001 [INF] Executed action "PeerReview.Controllers.HomeController.Index (PeerReview)" in 7870.9649ms (afa2e885)
2018-05-30T02:33:27.3281186+03:00 0HLE5NI5N86U9:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/bootstrap/dist/css/bootstrap.css   (ca22a1cb)
2018-05-30T02:33:27.3353863+03:00 0HLE5NI5N86UA:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/css/site.css   (ca22a1cb)
2018-05-30T02:33:27.3435838+03:00 0HLE5NI5N86U8:00000001 [INF] Request finished in 14489.5879ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T02:33:27.3489855+03:00 0HLE5NI5N86U8:00000002 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/jquery/dist/jquery.js   (ca22a1cb)
2018-05-30T02:33:27.3677389+03:00 0HLE5NI5N86U8:00000002 [INF] Sending file. Request path: '"/lib/jquery/dist/jquery.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/jquery/dist/jquery.js"' (27b0a520)
2018-05-30T02:33:27.3744557+03:00 0HLE5NI5N86UA:00000001 [INF] Sending file. Request path: '"/css/site.css"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/css/site.css"' (27b0a520)
2018-05-30T02:33:27.3752985+03:00 0HLE5NI5N86U9:00000001 [INF] Sending file. Request path: '"/lib/bootstrap/dist/css/bootstrap.css"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/bootstrap/dist/css/bootstrap.css"' (27b0a520)
2018-05-30T02:33:27.4000323+03:00 0HLE5NI5N86UA:00000001 [INF] Request finished in 64.7493ms 200 text/css (791a596a)
2018-05-30T02:33:27.4352348+03:00 0HLE5NI5N86U9:00000001 [INF] Request finished in 105.4139ms 200 text/css (791a596a)
2018-05-30T02:33:27.4401302+03:00 0HLE5NI5N86UA:00000002 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/js/site.js?v=7mkNbU1tgQL1bUeZe3j2R151hKLhLDKO4BBaR-iqCy0   (ca22a1cb)
2018-05-30T02:33:27.4422400+03:00 0HLE5NI5N86UA:00000002 [INF] Sending file. Request path: '"/js/site.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/js/site.js"' (27b0a520)
2018-05-30T02:33:27.4466587+03:00 0HLE5NI5N86UA:00000002 [INF] Request finished in 7.4885ms 200 application/javascript (791a596a)
2018-05-30T02:33:27.4469106+03:00 0HLE5NI5N86U9:00000002 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/bootstrap/dist/js/bootstrap.js   (ca22a1cb)
2018-05-30T02:33:27.4479377+03:00 0HLE5NI5N86UB:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/images/banner1.svg   (ca22a1cb)
2018-05-30T02:33:27.4508537+03:00 0HLE5NI5N86UC:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/images/banner2.svg   (ca22a1cb)
2018-05-30T02:33:27.4519167+03:00 0HLE5NI5N86UB:00000001 [INF] Sending file. Request path: '"/images/banner1.svg"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/images/banner1.svg"' (27b0a520)
2018-05-30T02:33:27.4524763+03:00 0HLE5NI5N86UD:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/images/banner3.svg   (ca22a1cb)
2018-05-30T02:33:27.4527992+03:00 0HLE5NI5N86UC:00000001 [INF] Sending file. Request path: '"/images/banner2.svg"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/images/banner2.svg"' (27b0a520)
2018-05-30T02:33:27.4532570+03:00 0HLE5NI5N86UD:00000001 [INF] Sending file. Request path: '"/images/banner3.svg"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/images/banner3.svg"' (27b0a520)
2018-05-30T02:33:27.4595162+03:00 0HLE5NI5N86U9:00000002 [INF] Sending file. Request path: '"/lib/bootstrap/dist/js/bootstrap.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/bootstrap/dist/js/bootstrap.js"' (27b0a520)
2018-05-30T02:33:27.4641271+03:00 0HLE5NI5N86UA:00000003 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/images/banner4.svg   (ca22a1cb)
2018-05-30T02:33:27.4645995+03:00 0HLE5NI5N86UA:00000003 [INF] Sending file. Request path: '"/images/banner4.svg"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/images/banner4.svg"' (27b0a520)
2018-05-30T02:33:27.4869566+03:00 0HLE5NI5N86UA:00000003 [INF] Request finished in 22.9696ms 200 image/svg+xml (791a596a)
2018-05-30T02:33:27.4870589+03:00 0HLE5NI5N86UC:00000001 [INF] Request finished in 36.4863ms 200 image/svg+xml (791a596a)
2018-05-30T02:33:27.4875244+03:00 0HLE5NI5N86UB:00000001 [INF] Request finished in 39.7921ms 200 image/svg+xml (791a596a)
2018-05-30T02:33:27.4889801+03:00 0HLE5NI5N86UD:00000001 [INF] Request finished in 37.0663ms 200 image/svg+xml (791a596a)
2018-05-30T02:33:27.5127694+03:00 0HLE5NI5N86UD:00000002 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2   (ca22a1cb)
2018-05-30T02:33:27.5132463+03:00 0HLE5NI5N86UD:00000002 [INF] Sending file. Request path: '"/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2"' (27b0a520)
2018-05-30T02:33:27.5217704+03:00 0HLE5NI5N86U8:00000002 [INF] Request finished in 172.3698ms 200 application/javascript (791a596a)
2018-05-30T02:33:27.5256888+03:00 0HLE5NI5N86U9:00000002 [INF] Request finished in 81.5179ms 200 application/javascript (791a596a)
2018-05-30T02:33:27.5338818+03:00 0HLE5NI5N86UD:00000002 [INF] Request finished in 21.2723ms 200 font/woff2 (791a596a)
2018-05-30T02:33:27.8561158+03:00 0HLE5NI5N86UD:00000003 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/favicon.ico   (ca22a1cb)
2018-05-30T02:33:27.8565672+03:00 0HLE5NI5N86UD:00000003 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/favicon.ico"' (27b0a520)
2018-05-30T02:33:27.8952687+03:00 0HLE5NI5N86UD:00000003 [INF] Request finished in 39.2113ms 200 image/x-icon (791a596a)
2018-05-30T02:35:26.5636111+03:00 0HLE5NI5N86UD:00000004 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T02:35:26.5812076+03:00 0HLE5NI5N86UD:00000004 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T02:35:26.6340067+03:00 0HLE5NI5N86UD:00000004 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T02:35:26.6458774+03:00 0HLE5NI5N86UD:00000004 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T02:35:26.6640023+03:00 0HLE5NI5N86UD:00000004 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T02:35:26.6703171+03:00 0HLE5NI5N86UD:00000004 [INF] Executed DbCommand ("2"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"CountInvites\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T02:35:27.5187869+03:00 0HLE5NI5N86UD:00000004 [INF] Executing ViewResult, running view at path "/Views/Manage/Index.cshtml". (9707eebe)
2018-05-30T02:35:30.2350549+03:00 0HLE5NI5N86UD:00000004 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 3620.8072ms (afa2e885)
2018-05-30T02:35:30.2379279+03:00 0HLE5NI5N86UD:00000004 [INF] Request finished in 3674.507ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T02:35:30.2618772+03:00 0HLE5NI5N86UD:00000005 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/jquery-validation/dist/jquery.validate.js   (ca22a1cb)
2018-05-30T02:35:30.2623724+03:00 0HLE5NI5N86UD:00000005 [INF] Sending file. Request path: '"/lib/jquery-validation/dist/jquery.validate.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/jquery-validation/dist/jquery.validate.js"' (27b0a520)
2018-05-30T02:35:30.2653090+03:00 0HLE5NI5N86UE:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js   (ca22a1cb)
2018-05-30T02:35:30.2659207+03:00 0HLE5NI5N86UE:00000001 [INF] Sending file. Request path: '"/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"' (27b0a520)
2018-05-30T02:35:30.3020871+03:00 0HLE5NI5N86UE:00000001 [INF] Request finished in 37.5311ms 200 application/javascript (791a596a)
2018-05-30T02:35:30.3215754+03:00 0HLE5NI5N86UD:00000005 [INF] Request finished in 59.8535ms 200 application/javascript (791a596a)
2018-05-30T02:59:19.4444259+03:00 0HLE5O0OH3R12:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T02:59:20.5172411+03:00 0HLE5O0OH3R12:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T02:59:20.8612667+03:00 0HLE5O0OH3R13:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T02:59:20.8673183+03:00 0HLE5O0OH3R13:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T02:59:21.4953417+03:00 0HLE5O0OH3R12:00000001 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T02:59:21.4953416+03:00 0HLE5O0OH3R13:00000001 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T02:59:21.5591542+03:00 0HLE5O0OH3R12:00000001 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T02:59:21.5591540+03:00 0HLE5O0OH3R13:00000001 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T02:59:21.5843485+03:00 0HLE5O0OH3R12:00000001 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 107.8534ms (afa2e885)
2018-05-30T02:59:21.9088016+03:00 0HLE5O0OH3R12:00000001 [ERR] An unhandled exception has occurred while executing the request (cf926119)
System.OperationCanceledException: The operation was canceled.
   at System.Threading.CancellationToken.ThrowOperationCanceledException()
   at Microsoft.AspNetCore.Identity.EntityFrameworkCore.UserStore`9.FindByIdAsync(String userId, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Identity.UserManager`1.GetUserAsync(ClaimsPrincipal principal)
   at PeerReview.Controllers.ManageController.<Index>d__12.MoveNext() in /home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/Controllers/ManageController.cs:line 56
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.MigrationsEndPointMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()
2018-05-30T02:59:22.1664843+03:00 0HLE5O0OH3R12:00000001 [INF] Request finished in 2747.8958ms 500 text/html; charset=utf-8 (791a596a)
2018-05-30T02:59:23.8014962+03:00 0HLE5O0OH3R13:00000001 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T02:59:28.1088914+03:00 0HLE5O0OH3R13:00000001 [ERR] Failed executing DbCommand ("507"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (627a98df)
Npgsql.PostgresException (0x80004005): 42703: столбец e.InvitesCount не существует
   at Npgsql.NpgsqlConnector.<DoReadMessage>d__157.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadExpecting>d__163`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlDataReader.<NextResult>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlDataReader.<<NextResultAsync>b__31_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlCommand.<Execute>d__71.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<ExecuteDbDataReader>d__92.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<>c__DisplayClass90_0.<<ExecuteDbDataReaderAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
2018-05-30T02:59:28.2776606+03:00 0HLE5O0OH3R13:00000001 [ERR] An exception occurred in the database while iterating the results of a query for context type '"PeerReview.Models.AppDbContext"'."
""Npgsql.PostgresException (0x80004005): 42703: столбец e.InvitesCount не существует
   at Npgsql.NpgsqlConnector.<DoReadMessage>d__157.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadExpecting>d__163`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlDataReader.<NextResult>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlDataReader.<<NextResultAsync>b__31_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlCommand.<Execute>d__71.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<ExecuteDbDataReader>d__92.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<>c__DisplayClass90_0.<<ExecuteDbDataReaderAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<BufferlessMoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.NpgsqlExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<MoveNext>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.<FirstOrDefault_>d__165`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.<MoveNext>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.<MoveNextCore>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.AsyncIterator`1.<MoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.<MoveNext>d__5.MoveNext()" (64e2593c)
Npgsql.PostgresException (0x80004005): 42703: столбец e.InvitesCount не существует
   at Npgsql.NpgsqlConnector.<DoReadMessage>d__157.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadExpecting>d__163`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlDataReader.<NextResult>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlDataReader.<<NextResultAsync>b__31_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlCommand.<Execute>d__71.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<ExecuteDbDataReader>d__92.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<>c__DisplayClass90_0.<<ExecuteDbDataReaderAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<BufferlessMoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.NpgsqlExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<MoveNext>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.<FirstOrDefault_>d__165`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.<MoveNext>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.<MoveNextCore>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.AsyncIterator`1.<MoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.<MoveNext>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<ExecuteSingletonAsyncQuery>d__23`1.MoveNext()
2018-05-30T02:59:28.3468992+03:00 0HLE5O0OH3R13:00000001 [ERR] An exception occurred in the database while iterating the results of a query for context type '"PeerReview.Models.AppDbContext"'."
""Npgsql.PostgresException (0x80004005): 42703: столбец e.InvitesCount не существует
   at Npgsql.NpgsqlConnector.<DoReadMessage>d__157.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadExpecting>d__163`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlDataReader.<NextResult>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlDataReader.<<NextResultAsync>b__31_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlCommand.<Execute>d__71.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<ExecuteDbDataReader>d__92.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<>c__DisplayClass90_0.<<ExecuteDbDataReaderAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<BufferlessMoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.NpgsqlExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<MoveNext>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.<FirstOrDefault_>d__165`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.<MoveNext>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.<MoveNextCore>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.AsyncIterator`1.<MoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.<MoveNext>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<ExecuteSingletonAsyncQuery>d__23`1.MoveNext()" (64e2593c)
Npgsql.PostgresException (0x80004005): 42703: столбец e.InvitesCount не существует
   at Npgsql.NpgsqlConnector.<DoReadMessage>d__157.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Npgsql.NpgsqlConnector.<ReadMessage>d__156.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlConnector.<ReadExpecting>d__163`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlDataReader.<NextResult>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlDataReader.<<NextResultAsync>b__31_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Npgsql.NpgsqlCommand.<Execute>d__71.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<ExecuteDbDataReader>d__92.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Runtime.CompilerServices.ValueTaskAwaiter`1.GetResult()
   at Npgsql.NpgsqlCommand.<>c__DisplayClass90_0.<<ExecuteDbDataReaderAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<BufferlessMoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.NpgsqlExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.<MoveNext>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.<FirstOrDefault_>d__165`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.TaskResultAsyncEnumerable`1.Enumerator.<MoveNext>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.SelectEnumerableAsyncIterator`2.<MoveNextCore>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Linq.AsyncEnumerable.AsyncIterator`1.<MoveNext>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.<MoveNext>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<ExecuteSingletonAsyncQuery>d__23`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at PeerReview.Controllers.ManageController.<Index>d__12.MoveNext() in /home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/Controllers/ManageController.cs:line 56
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.MigrationsEndPointMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.<Invoke>d__6.MoveNext()
2018-05-30T02:59:28.3538045+03:00 0HLE5O0OH3R13:00000001 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 6892.6848ms (afa2e885)
2018-05-30T02:59:28.7934822+03:00 0HLE5O0OH3R13:00000001 [INF] Executed DbCommand ("63"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT EXISTS (SELECT 1 FROM pg_catalog.pg_class c JOIN pg_catalog.pg_namespace n ON n.oid=c.relnamespace WHERE c.relname='__EFMigrationsHistory');" (0723d8ff)
2018-05-30T02:59:28.9102379+03:00 0HLE5O0OH3R13:00000001 [INF] Executed DbCommand ("102"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT \"MigrationId\", \"ProductVersion\"
FROM \"__EFMigrationsHistory\"
ORDER BY \"MigrationId\";" (0723d8ff)
2018-05-30T02:59:28.9471016+03:00 0HLE5O0OH3R13:00000001 [INF] Request finished in 8085.1292ms 500 text/html; charset=utf-8 (791a596a)
2018-05-30T03:00:17.0237020+03:00  [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:03:37.7140303+03:00  [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:03:38.4402881+03:00  [INF] Executed DbCommand ("65"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT EXISTS (SELECT 1 FROM pg_catalog.pg_class c JOIN pg_catalog.pg_namespace n ON n.oid=c.relnamespace WHERE c.relname='__EFMigrationsHistory');" (0723d8ff)
2018-05-30T03:03:38.4998693+03:00  [INF] Executed DbCommand ("32"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT EXISTS (SELECT 1 FROM pg_catalog.pg_class c JOIN pg_catalog.pg_namespace n ON n.oid=c.relnamespace WHERE c.relname='__EFMigrationsHistory');" (0723d8ff)
2018-05-30T03:03:38.5226247+03:00  [INF] Executed DbCommand ("5"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""SELECT \"MigrationId\", \"ProductVersion\"
FROM \"__EFMigrationsHistory\"
ORDER BY \"MigrationId\";" (0723d8ff)
2018-05-30T03:03:38.5632640+03:00  [INF] Applying migration '"20180530000017_upp15"'. (0e2ac64d)
2018-05-30T03:03:38.8511688+03:00  [INF] Executed DbCommand ("206"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""ALTER TABLE \"AspNetUsers\" RENAME COLUMN \"CountInvites\" TO \"InvitesCount\";" (0723d8ff)
2018-05-30T03:03:38.8524281+03:00  [INF] Executed DbCommand ("0"ms) [Parameters=[""], CommandType='Text', CommandTimeout='30']"
""INSERT INTO \"__EFMigrationsHistory\" (\"MigrationId\", \"ProductVersion\")
VALUES ('20180530000017_upp15', '2.0.3-rtm-10026');" (0723d8ff)
2018-05-30T03:07:17.8101973+03:00 0HLE5O575JGTJ:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T03:07:19.8918253+03:00 0HLE5O575JGTJ:00000001 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:07:21.7252209+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("74"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:07:22.4002786+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("21"ms) [Parameters=["@__user_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"uc\".\"Id\", \"uc\".\"ClaimType\", \"uc\".\"ClaimValue\", \"uc\".\"UserId\"
FROM \"AspNetUserClaims\" AS \"uc\"
WHERE \"uc\".\"UserId\" = @__user_Id_0" (0723d8ff)
2018-05-30T03:07:22.5352047+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("4"ms) [Parameters=["@__userId_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"role\".\"Name\"
FROM \"AspNetUserRoles\" AS \"userRole\"
INNER JOIN \"AspNetRoles\" AS \"role\" ON \"userRole\".\"RoleId\" = \"role\".\"Id\"
WHERE \"userRole\".\"UserId\" = @__userId_0" (0723d8ff)
2018-05-30T03:07:22.5979022+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("9"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T03:07:22.6741832+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T03:07:22.6934361+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T03:07:22.7008826+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("3"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T03:07:22.7035502+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("1"ms) [Parameters=["@__normalizedName_0='?' (Size = 256)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"r\".\"Id\", \"r\".\"ConcurrencyStamp\", \"r\".\"Name\", \"r\".\"NormalizedName\"
FROM \"AspNetRoles\" AS \"r\"
WHERE \"r\".\"NormalizedName\" = @__normalizedName_0
LIMIT 1" (0723d8ff)
2018-05-30T03:07:22.7063108+03:00 0HLE5O575JGTJ:00000001 [INF] Executed DbCommand ("0"ms) [Parameters=["@__role_Id_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"rc\".\"ClaimType\", \"rc\".\"ClaimValue\"
FROM \"AspNetRoleClaims\" AS \"rc\"
WHERE \"rc\".\"RoleId\" = @__role_Id_0" (0723d8ff)
2018-05-30T03:07:22.7161226+03:00 0HLE5O575JGTJ:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:07:22.9354637+03:00 0HLE5O575JGTJ:00000001 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:07:22.9631440+03:00 0HLE5O575JGTJ:00000001 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:07:29.1182652+03:00 0HLE5O575JGTJ:00000001 [INF] Executing ViewResult, running view at path "/Views/Manage/Index.cshtml". (9707eebe)
2018-05-30T03:07:33.7931971+03:00 0HLE5O575JGTJ:00000001 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 10865.2461ms (afa2e885)
2018-05-30T03:07:33.8107924+03:00 0HLE5O575JGTJ:00000001 [INF] Request finished in 16025.8982ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:15:10.2247407+03:00 0HLE5O9JUVPJR:00000001 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/CreateInvite   (ca22a1cb)
2018-05-30T03:15:10.7748356+03:00 0HLE5O9JUVPJR:00000001 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:15:10.9891448+03:00 0HLE5O9JUVPJR:00000001 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:15:11.0170243+03:00 0HLE5O9JUVPJR:00000001 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:15:12.5370814+03:00 0HLE5O9JUVPJR:00000001 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:15:14.0115207+03:00 0HLE5O9JUVPJR:00000001 [INF] Executed DbCommand ("66"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:15:18.5350384+03:00 0HLE5O9JUVPJR:00000001 [INF] Executing ViewResult, running view at path "/Views/Manage/CreateInvite.cshtml". (9707eebe)
2018-05-30T03:15:20.9477033+03:00 0HLE5O9JUVPJR:00000001 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 9965.8076ms (afa2e885)
2018-05-30T03:15:20.9666199+03:00 0HLE5O9JUVPJR:00000001 [INF] Request finished in 10767.8214ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:15:21.1942141+03:00 0HLE5O9JUVPJR:00000002 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/favicon.ico   (ca22a1cb)
2018-05-30T03:15:21.2254843+03:00 0HLE5O9JUVPJR:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/favicon.ico"' (27b0a520)
2018-05-30T03:15:21.2438481+03:00 0HLE5O9JUVPJR:00000002 [INF] Request finished in 49.8265ms 200 image/x-icon (791a596a)
2018-05-30T03:15:36.6126401+03:00 0HLE5O9JUVPJR:00000003 [INF] Request starting HTTP/1.1 POST http://127.0.0.1:5000/Manage/CreateInvite application/x-www-form-urlencoded 257 (ca22a1cb)
2018-05-30T03:15:36.6239671+03:00 0HLE5O9JUVPJR:00000003 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:15:36.6842319+03:00 0HLE5O9JUVPJR:00000003 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:15:36.8084136+03:00 0HLE5O9JUVPJR:00000003 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (["PeerReview.ViewModels.ManageViewModel.CreateInviteViewModel"]) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:15:36.8202751+03:00 0HLE5O9JUVPJR:00000003 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:15:36.9757133+03:00 0HLE5O9JUVPJR:00000003 [INF] Executed DbCommand ("3"ms) [Parameters=["@__p_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT CASE
    WHEN @__p_0 IN (
        SELECT \"x\".\"Email\"
        FROM \"AspNetUsers\" AS \"x\"
    )
    THEN TRUE::bool ELSE FALSE::bool
END" (0723d8ff)
2018-05-30T03:15:36.9879744+03:00 0HLE5O9JUVPJR:00000003 [INF] Executing RedirectResult, redirecting to "/Manage/Index". (d98d540e)
2018-05-30T03:15:36.9919429+03:00 0HLE5O9JUVPJR:00000003 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 308.1848ms (afa2e885)
2018-05-30T03:15:36.9971296+03:00 0HLE5O9JUVPJR:00000003 [INF] Request finished in 384.6018ms 302  (791a596a)
2018-05-30T03:15:37.0020460+03:00 0HLE5O9JUVPJR:00000004 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T03:15:37.0033699+03:00 0HLE5O9JUVPJR:00000004 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:15:37.0151378+03:00 0HLE5O9JUVPJR:00000004 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:15:37.0157439+03:00 0HLE5O9JUVPJR:00000004 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:15:37.0196099+03:00 0HLE5O9JUVPJR:00000004 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:15:37.0368240+03:00 0HLE5O9JUVPJR:00000004 [INF] Executed DbCommand ("3"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:15:37.6432402+03:00 0HLE5O9JUVPJR:00000004 [INF] Executing ViewResult, running view at path "/Views/Manage/Index.cshtml". (9707eebe)
2018-05-30T03:15:38.4927925+03:00 0HLE5O9JUVPJR:00000004 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 1477.493ms (afa2e885)
2018-05-30T03:15:38.4950098+03:00 0HLE5O9JUVPJR:00000004 [INF] Request finished in 1492.3524ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:17:11.9482151+03:00 0HLE5O9JUVPJR:00000005 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/CreateInvite   (ca22a1cb)
2018-05-30T03:17:11.9511263+03:00 0HLE5O9JUVPJR:00000005 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:11.9519657+03:00 0HLE5O9JUVPJR:00000005 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:11.9524821+03:00 0HLE5O9JUVPJR:00000005 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:11.9584947+03:00 0HLE5O9JUVPJR:00000005 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:11.9706986+03:00 0HLE5O9JUVPJR:00000005 [INF] Executed DbCommand ("6"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:11.9741390+03:00 0HLE5O9JUVPJR:00000005 [INF] Executing ViewResult, running view at path "/Views/Manage/CreateInvite.cshtml". (9707eebe)
2018-05-30T03:17:11.9801246+03:00 0HLE5O9JUVPJR:00000005 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 27.6888ms (afa2e885)
2018-05-30T03:17:11.9842821+03:00 0HLE5O9JUVPJR:00000005 [INF] Request finished in 35.0562ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:17:15.3278543+03:00 0HLE5O9JUVPJR:00000006 [INF] Request starting HTTP/1.1 POST http://127.0.0.1:5000/Manage/CreateInvite application/x-www-form-urlencoded 254 (ca22a1cb)
2018-05-30T03:17:15.3291395+03:00 0HLE5O9JUVPJR:00000006 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:15.3296539+03:00 0HLE5O9JUVPJR:00000006 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:15.3324709+03:00 0HLE5O9JUVPJR:00000006 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (["PeerReview.ViewModels.ManageViewModel.CreateInviteViewModel"]) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:15.3340359+03:00 0HLE5O9JUVPJR:00000006 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:15.3400732+03:00 0HLE5O9JUVPJR:00000006 [INF] Executed DbCommand ("1"ms) [Parameters=["@__p_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT CASE
    WHEN @__p_0 IN (
        SELECT \"x\".\"Email\"
        FROM \"AspNetUsers\" AS \"x\"
    )
    THEN TRUE::bool ELSE FALSE::bool
END" (0723d8ff)
2018-05-30T03:17:15.3811114+03:00 0HLE5O9JUVPJR:00000006 [INF] Executed DbCommand ("1"ms) [Parameters=["@__GetUserId_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"x\".\"Id\", \"x\".\"AccessFailedCount\", \"x\".\"ConcurrencyStamp\", \"x\".\"Email\", \"x\".\"EmailConfirmed\", \"x\".\"InvitesCount\", \"x\".\"LockoutEnabled\", \"x\".\"LockoutEnd\", \"x\".\"NormalizedEmail\", \"x\".\"NormalizedUserName\", \"x\".\"PasswordHash\", \"x\".\"PhoneNumber\", \"x\".\"PhoneNumberConfirmed\", \"x\".\"SecurityStamp\", \"x\".\"TwoFactorEnabled\", \"x\".\"UserName\"
FROM \"AspNetUsers\" AS \"x\"
WHERE \"x\".\"Id\" = @__GetUserId_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:15.7596000+03:00 0HLE5O9JUVPJR:00000006 [INF] Executed DbCommand ("7"ms) [Parameters=["@p0='?', @p1='?', @p2='?'"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO \"Invites\" (\"InviteCode\", \"UserId\", \"UserId1\")
VALUES (@p0, @p1, @p2)
RETURNING \"Id\";" (0723d8ff)
2018-05-30T03:17:18.1999695+03:00 0HLE5O9JUVPJR:00000006 [INF] Executing RedirectResult, redirecting to "/Manage/Index". (d98d540e)
2018-05-30T03:17:18.2006107+03:00 0HLE5O9JUVPJR:00000006 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 2870.9922ms (afa2e885)
2018-05-30T03:17:18.2024185+03:00 0HLE5O9JUVPJR:00000006 [INF] Request finished in 2874.292ms 302  (791a596a)
2018-05-30T03:17:18.2051380+03:00 0HLE5O9JUVPJR:00000007 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T03:17:18.2062247+03:00 0HLE5O9JUVPJR:00000007 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:18.2067549+03:00 0HLE5O9JUVPJR:00000007 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:18.2073834+03:00 0HLE5O9JUVPJR:00000007 [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:18.2082701+03:00 0HLE5O9JUVPJR:00000007 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:18.2135345+03:00 0HLE5O9JUVPJR:00000007 [INF] Executed DbCommand ("3"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:18.2145489+03:00 0HLE5O9JUVPJR:00000007 [INF] Executing ViewResult, running view at path "/Views/Manage/Index.cshtml". (9707eebe)
2018-05-30T03:17:18.2193251+03:00 0HLE5O9JUVPJR:00000007 [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 12.5484ms (afa2e885)
2018-05-30T03:17:18.2225325+03:00 0HLE5O9JUVPJR:00000007 [INF] Request finished in 17.4035ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:17:36.2898624+03:00 0HLE5O9JUVPJR:00000008 [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/CreateInvite   (ca22a1cb)
2018-05-30T03:17:36.2910549+03:00 0HLE5O9JUVPJR:00000008 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:36.2929006+03:00 0HLE5O9JUVPJR:00000008 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:36.2934306+03:00 0HLE5O9JUVPJR:00000008 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:36.2944584+03:00 0HLE5O9JUVPJR:00000008 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:36.2993965+03:00 0HLE5O9JUVPJR:00000008 [INF] Executed DbCommand ("1"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:36.3003782+03:00 0HLE5O9JUVPJR:00000008 [INF] Executing ViewResult, running view at path "/Views/Manage/CreateInvite.cshtml". (9707eebe)
2018-05-30T03:17:36.3039290+03:00 0HLE5O9JUVPJR:00000008 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 11.128ms (afa2e885)
2018-05-30T03:17:36.3045715+03:00 0HLE5O9JUVPJR:00000008 [INF] Request finished in 15.0476ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:17:40.6354503+03:00 0HLE5O9JUVPJR:00000009 [INF] Request starting HTTP/1.1 POST http://127.0.0.1:5000/Manage/CreateInvite application/x-www-form-urlencoded 254 (ca22a1cb)
2018-05-30T03:17:40.6364194+03:00 0HLE5O9JUVPJR:00000009 [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:40.6369069+03:00 0HLE5O9JUVPJR:00000009 [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:40.6378018+03:00 0HLE5O9JUVPJR:00000009 [INF] Executing action method "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" with arguments (["PeerReview.ViewModels.ManageViewModel.CreateInviteViewModel"]) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:40.6387484+03:00 0HLE5O9JUVPJR:00000009 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:40.6427265+03:00 0HLE5O9JUVPJR:00000009 [INF] Executed DbCommand ("1"ms) [Parameters=["@__p_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT CASE
    WHEN @__p_0 IN (
        SELECT \"x\".\"Email\"
        FROM \"AspNetUsers\" AS \"x\"
    )
    THEN TRUE::bool ELSE FALSE::bool
END" (0723d8ff)
2018-05-30T03:17:40.6500968+03:00 0HLE5O9JUVPJR:00000009 [INF] Executed DbCommand ("1"ms) [Parameters=["@__GetUserId_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"x\".\"Id\", \"x\".\"AccessFailedCount\", \"x\".\"ConcurrencyStamp\", \"x\".\"Email\", \"x\".\"EmailConfirmed\", \"x\".\"InvitesCount\", \"x\".\"LockoutEnabled\", \"x\".\"LockoutEnd\", \"x\".\"NormalizedEmail\", \"x\".\"NormalizedUserName\", \"x\".\"PasswordHash\", \"x\".\"PhoneNumber\", \"x\".\"PhoneNumberConfirmed\", \"x\".\"SecurityStamp\", \"x\".\"TwoFactorEnabled\", \"x\".\"UserName\"
FROM \"AspNetUsers\" AS \"x\"
WHERE \"x\".\"Id\" = @__GetUserId_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:40.6579627+03:00 0HLE5O9JUVPJR:00000009 [INF] Executed DbCommand ("1"ms) [Parameters=["@p0='?', @p1='?', @p2='?'"], CommandType='Text', CommandTimeout='30']"
""INSERT INTO \"Invites\" (\"InviteCode\", \"UserId\", \"UserId1\")
VALUES (@p0, @p1, @p2)
RETURNING \"Id\";" (0723d8ff)
2018-05-30T03:17:41.9296918+03:00 0HLE5O9JUVPJR:00000009 [INF] Executing RedirectResult, redirecting to "/Manage/Index". (d98d540e)
2018-05-30T03:17:41.9304760+03:00 0HLE5O9JUVPJR:00000009 [INF] Executed action "PeerReview.Controllers.ManageController.CreateInvite (PeerReview)" in 1293.507ms (afa2e885)
2018-05-30T03:17:41.9311103+03:00 0HLE5O9JUVPJR:00000009 [INF] Request finished in 1295.8123ms 302  (791a596a)
2018-05-30T03:17:41.9359897+03:00 0HLE5O9JUVPJR:0000000A [INF] Request starting HTTP/1.1 GET http://127.0.0.1:5000/Manage/Index   (ca22a1cb)
2018-05-30T03:17:41.9371359+03:00 0HLE5O9JUVPJR:0000000A [INF] AuthenticationScheme: "Identity.Application" was successfully authenticated. (1805f3b3)
2018-05-30T03:17:41.9376753+03:00 0HLE5O9JUVPJR:0000000A [INF] Authorization was successful for user: "m.k-mister@yandex.ru". (d7eb274a)
2018-05-30T03:17:41.9384796+03:00 0HLE5O9JUVPJR:0000000A [INF] Executing action method "PeerReview.Controllers.ManageController.Index (PeerReview)" with arguments (null) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:17:41.9396772+03:00 0HLE5O9JUVPJR:0000000A [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:17:41.9446088+03:00 0HLE5O9JUVPJR:0000000A [INF] Executed DbCommand ("1"ms) [Parameters=["@__get_Item_0='?'"], CommandType='Text', CommandTimeout='30']"
""SELECT \"e\".\"Id\", \"e\".\"AccessFailedCount\", \"e\".\"ConcurrencyStamp\", \"e\".\"Email\", \"e\".\"EmailConfirmed\", \"e\".\"InvitesCount\", \"e\".\"LockoutEnabled\", \"e\".\"LockoutEnd\", \"e\".\"NormalizedEmail\", \"e\".\"NormalizedUserName\", \"e\".\"PasswordHash\", \"e\".\"PhoneNumber\", \"e\".\"PhoneNumberConfirmed\", \"e\".\"SecurityStamp\", \"e\".\"TwoFactorEnabled\", \"e\".\"UserName\"
FROM \"AspNetUsers\" AS \"e\"
WHERE \"e\".\"Id\" = @__get_Item_0
LIMIT 1" (0723d8ff)
2018-05-30T03:17:41.9455858+03:00 0HLE5O9JUVPJR:0000000A [INF] Executing ViewResult, running view at path "/Views/Manage/Index.cshtml". (9707eebe)
2018-05-30T03:17:41.9512912+03:00 0HLE5O9JUVPJR:0000000A [INF] Executed action "PeerReview.Controllers.ManageController.Index (PeerReview)" in 12.9212ms (afa2e885)
2018-05-30T03:17:41.9518929+03:00 0HLE5O9JUVPJR:0000000A [INF] Request finished in 16.1609ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:18:19.4964257+03:00 0HLE5O9JUVPJS:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Account/Register/f13a0cfdca59a62fd50f243eb0f8d3863b9adadf5e0cdd74e3f04bdd8fa61750   (ca22a1cb)
2018-05-30T03:18:19.5089032+03:00 0HLE5O9JUVPJS:00000001 [INF] Executing action method "PeerReview.Controllers.AccountController.Register (PeerReview)" with arguments (["f13a0cfdca59a62fd50f243eb0f8d3863b9adadf5e0cdd74e3f04bdd8fa61750"]) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:18:20.0635858+03:00 0HLE5O9JUVPJS:00000001 [INF] Executing ViewResult, running view at path "/Views/Account/Register.cshtml". (9707eebe)
2018-05-30T03:18:20.1433019+03:00 0HLE5O9JUVPJS:00000001 [INF] Executed action "PeerReview.Controllers.AccountController.Register (PeerReview)" in 637.3611ms (afa2e885)
2018-05-30T03:18:20.1448187+03:00 0HLE5O9JUVPJS:00000001 [INF] Request finished in 648.5678ms 200 text/html; charset=utf-8 (791a596a)
2018-05-30T03:18:20.1709320+03:00 0HLE5O9JUVPJT:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/site.css   (ca22a1cb)
2018-05-30T03:18:20.1718535+03:00 0HLE5O9JUVPJT:00000001 [INF] Sending file. Request path: '"/css/site.css"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/css/site.css"' (27b0a520)
2018-05-30T03:18:20.1747904+03:00 0HLE5O9JUVPJS:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/lib/bootstrap/dist/css/bootstrap.css   (ca22a1cb)
2018-05-30T03:18:20.1747129+03:00 0HLE5O9JUVPJT:00000001 [INF] Request finished in 3.9931ms 200 text/css (791a596a)
2018-05-30T03:18:20.1752685+03:00 0HLE5O9JUVPK0:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/site.js?v=7mkNbU1tgQL1bUeZe3j2R151hKLhLDKO4BBaR-iqCy0   (ca22a1cb)
2018-05-30T03:18:20.1751386+03:00 0HLE5O9JUVPJV:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/lib/jquery/dist/jquery.js   (ca22a1cb)
2018-05-30T03:18:20.1754703+03:00 0HLE5O9JUVPJU:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/lib/bootstrap/dist/js/bootstrap.js   (ca22a1cb)
2018-05-30T03:18:20.1757698+03:00 0HLE5O9JUVPJV:00000001 [INF] Sending file. Request path: '"/lib/jquery/dist/jquery.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/jquery/dist/jquery.js"' (27b0a520)
2018-05-30T03:18:20.1758937+03:00 0HLE5O9JUVPJU:00000001 [INF] Sending file. Request path: '"/lib/bootstrap/dist/js/bootstrap.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/bootstrap/dist/js/bootstrap.js"' (27b0a520)
2018-05-30T03:18:20.1763872+03:00 0HLE5O9JUVPK0:00000001 [INF] Sending file. Request path: '"/js/site.js"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/js/site.js"' (27b0a520)
2018-05-30T03:18:20.1769214+03:00 0HLE5O9JUVPJS:00000002 [INF] Sending file. Request path: '"/lib/bootstrap/dist/css/bootstrap.css"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/lib/bootstrap/dist/css/bootstrap.css"' (27b0a520)
2018-05-30T03:18:20.2040397+03:00 0HLE5O9JUVPJU:00000001 [INF] Request finished in 29.2446ms 200 application/javascript (791a596a)
2018-05-30T03:18:20.2084079+03:00 0HLE5O9JUVPJV:00000001 [INF] Request finished in 33.3585ms 200 application/javascript (791a596a)
2018-05-30T03:18:20.2156985+03:00 0HLE5O9JUVPK0:00000001 [INF] Request finished in 40.3946ms 200 application/javascript (791a596a)
2018-05-30T03:18:20.2234873+03:00 0HLE5O9JUVPJS:00000002 [INF] Request finished in 48.7973ms 200 text/css (791a596a)
2018-05-30T03:18:20.4294535+03:00 0HLE5O9JUVPJS:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-05-30T03:18:20.4299474+03:00 0HLE5O9JUVPJS:00000003 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"/home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/wwwroot/favicon.ico"' (27b0a520)
2018-05-30T03:18:20.4535004+03:00 0HLE5O9JUVPJS:00000003 [INF] Request finished in 24.163ms 200 image/x-icon (791a596a)
2018-05-30T03:18:49.9774980+03:00 0HLE5O9JUVPJS:00000004 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Account/Register/f13a0cfdca59a62fd50f243eb0f8d3863b9adadf5e0cdd74e3f04bdd8fa61750 application/x-www-form-urlencoded 327 (ca22a1cb)
2018-05-30T03:18:49.9979985+03:00 0HLE5O9JUVPJS:00000004 [INF] Executing action method "PeerReview.Controllers.AccountController.Register (PeerReview)" with arguments (["PeerReview.ViewModels.AccountViewModels.RegisterViewModel"]) - ModelState is Valid (ba7f4ac2)
2018-05-30T03:18:50.0077299+03:00 0HLE5O9JUVPJS:00000004 [INF] Entity Framework Core "2.0.3-rtm-10026" initialized '"AppDbContext"' using provider '"Npgsql.EntityFrameworkCore.PostgreSQL"' with options: "None" (9958f5bb)
2018-05-30T03:18:50.0192656+03:00 0HLE5O9JUVPJS:00000004 [INF] Executed action "PeerReview.Controllers.AccountController.Register (PeerReview)" in 26.8715ms (afa2e885)
2018-05-30T03:18:50.0961542+03:00 0HLE5O9JUVPJS:00000004 [ERR] An unhandled exception has occurred while executing the request (cf926119)
System.InvalidOperationException: A parameterless constructor was not found on entity type 'Invite'. In order to create an instance of 'Invite' EF requires that a parameterless constructor be declared.
   at Microsoft.EntityFrameworkCore.Metadata.Internal.EntityMaterializerSource.CreateMaterializeExpression(IEntityType entityType, Expression valueBufferExpression, Int32[] indexMap)
   at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.Internal.MaterializerFactory.CreateMaterializer(IEntityType entityType, SelectExpression selectExpression, Func`3 projectionAdder, IQuerySource querySource, Dictionary`2& typeIndexMap)
   at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.RelationalEntityQueryableExpressionVisitor.CreateShaper(Type elementType, IEntityType entityType, SelectExpression selectExpression)
   at Microsoft.EntityFrameworkCore.Query.ExpressionVisitors.RelationalEntityQueryableExpressionVisitor.VisitEntityQueryable(Type elementType)
   at System.Linq.Expressions.ConstantExpression.Accept(ExpressionVisitor visitor)
   at System.Linq.Expressions.ExpressionVisitor.Visit(Expression node)
   at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.ReplaceClauseReferences(Expression expression, IQuerySource querySource, Boolean inProjection)
   at Microsoft.EntityFrameworkCore.Query.RelationalQueryModelVisitor.CompileMainFromClauseExpression(MainFromClause mainFromClause, QueryModel queryModel)
   at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.VisitMainFromClause(MainFromClause fromClause, QueryModel queryModel)
   at Remotion.Linq.Clauses.MainFromClause.Accept(IQueryModelVisitor visitor, QueryModel queryModel)
   at Remotion.Linq.QueryModelVisitorBase.VisitQueryModel(QueryModel queryModel)
   at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.VisitQueryModel(QueryModel queryModel)
   at Microsoft.EntityFrameworkCore.Query.RelationalQueryModelVisitor.VisitQueryModel(QueryModel queryModel)
   at Microsoft.EntityFrameworkCore.Query.EntityQueryModelVisitor.CreateQueryExecutor[TResult](QueryModel queryModel)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.CompileQueryCore[TResult](Expression query, INodeTypeProvider nodeTypeProvider, IDatabase database, IDiagnosticsLogger`1 logger, Type contextType)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.<>c__DisplayClass15_0`1.<Execute>b__0()
   at Microsoft.EntityFrameworkCore.Query.Internal.CompiledQueryCache.GetOrAddQueryCore[TFunc](Object cacheKey, Func`1 compiler)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryCompiler.Execute[TResult](Expression query)
   at System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   at PeerReview.Controllers.AccountController.<Register>d__8.MoveNext() in /home/mister/Yandex.Disk/0_ВУЗ/2 КУРС/2_С#/PeerReview/PeerReview/Controllers/AccountController.cs:line 78
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.MigrationsEndPointMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.<Invoke>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()
2018-05-30T03:18:50.1964682+03:00 0HLE5O9JUVPJS:00000004 [INF] Request finished in 219.2258ms 500 text/html; charset=utf-8 (791a596a)
